kde4_add_unit_test() alredy calls add_test().

Avoids errors such as:
CMake Error at tests/CMakeLists.txt:9 (add_test):
  add_test given test name "testscreenconfig" which already exists in this
  directory.
Call Stack (most recent call first):
  tests/CMakeLists.txt:13 (KSCREEN_TESTS)
--- tests/CMakeLists.txt.orig	2017-03-26 22:46:03.553275000 +0200
+++ tests/CMakeLists.txt	2017-03-26 22:46:08.261918000 +0200
@@ -6,7 +6,7 @@
         include_directories(${QT_INCLUDES} ${KDE4_INCLUDES} ${CMAKE_CURRENT_BINARY_DIR} ${X11_Xrandr_INCLUDE_PATH})
         kde4_add_unit_test(${_testname} ${_testname}.cpp ${_testname}.moc)
         target_link_libraries(${_testname} ${QT_QTTEST_LIBRARY} ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} kscreen)
-        add_test(${_testname} ${CMAKE_CURRENT_BINARY_DIR}/${_testname})
+        # add_test(${_testname} ${CMAKE_CURRENT_BINARY_DIR}/${_testname})
     endforeach(_testname)
 endmacro(KSCREEN_TESTS)
 
